hydrolib/core/io/fnm/models.py

```diff
@@ -28,7 +29,7 @@ class RainfallRunoffModel(FileModel):
     pluvius: Path = Path("pluvius.3b")
     pluvius_general: Path = Path("pluvius.alg")
     kasklasse: Path = Path("kasklass")
-    bui_file: Path = Path("default.bui")
+    bui_file: Optional[BuiModel] = None
```

Comment:
Might need to check whether these fields (in the .fnm.file) always need to show the default filename, whether it exists or not. Some test changes are related to this.

Reply:
From my understanding if a default path is given but it does not exist it will try to create the model based on the current logic, so it will 'crash'. If no path is given then nothing should happen.